sdk/pinocchio/src/log.rs

```diff
@@ -28,13 +28,24 @@
 //! [`RpcClient::get_transaction`]: https://docs.rs/solana-rpc-client/latest/solana_rpc_client/rpc_client/struct.RpcClient.html#method.get_transaction
 //!
 //! While most logging functions are defined in this module, [`Pubkey`]s can
-//! also be efficiently logged with the [`Pubkey::log`] function.
+//! also be efficiently logged with the [`pubkey::log`] function.
 //!
 //! [`Pubkey`]: crate::pubkey::Pubkey
-//! [`Pubkey::log`]: crate::pubkey::Pubkey::log
+//! [`pubkey::log`]: crate::pubkey::log
 
 use crate::{account_info::AccountInfo, pubkey::log};
 
+/// Print a message to the log.
+///
+/// Supports simple strings of type `&str`. The expression will be passed
+/// directly to [`sol_log`]. This is typically used for logging static strings.
+///
+/// # Examples
+///
+/// ```
+/// use pinocchio::msg;
+///
+/// msg!("verifying multisig");
```

Comment:
```suggestion
/// ```
/// use pinocchio::msg;
///
/// msg!("verifying multisig");
/// ```
```